benchmark_utils/tuning_utils.py

```diff
@@ -26,14 +24,31 @@ def format_and_save_tuning_results(tuning_results, variable: str, training_datas
                     else:
                         results[key].append(np.nan)
         results = pd.DataFrame(results)
-        results["hyperparameter"] = hyperparameter
+        
+        hyperparameters = path.split("/")[6]
```

Comment:
I get this bug here:

![image](https://github.com/owkin/deepdeconv-fork/assets/48271623/5ea47f4f-e4f9-4339-b843-6cc71765be74)

Reply:
If you want to subset the string with hyperparameters (not the most reliable way tbh) and expect to always get it from the last string after the split, I suggest using `.split("/")[-1]`

Reply:
Yes I've changed it to "-1", thanks! Weird that you got an issue with it, when running it on my side, the 6th part is always the last one, so no problem...
Agreed that this way is not ideal overall, but I'm not sure how to get these information otherwise!